service/scaladsl/server/src/main/scala/com/lightbend/lagom/internal/scaladsl/server/ScaladslServiceRouter.scala

```diff
@@ -77,16 +78,24 @@ class ScaladslServiceRouter(override protected val descriptor: Descriptor, servi
     }
   }
 
+  private val config = ConfigFactory.load
+  private val default_log_level = config.getString("lagom.logging.default")
+  private val log_levels: Map[String, String] = config.getConfig("lagom.logging.exception.logging.level").root.unwrapped.asScala.toMap.mapValues(_.asInstanceOf[String])
```

Comment:
These should use normal Scala naming conventions (camelCase, instead of underscores).

https://docs.scala-lang.org/style/naming-conventions.html

Reply:
Also, we're trying to model all sets of configuration settings into small, better typed classes. See for example the [`ConsumerConfig`](https://github.com/lagom/lagom/blob/55c5d1a92edf89c8ec2bd39b959e889c919627ee/service/core/kafka/client/src/main/scala/com/lightbend/lagom/internal/broker/kafka/KafkaConfig.scala#L60-L82)

It'd be great to isolate into a separate class the loading of settings (given an injected `Config` as mentioned by @TimMoore above).